Tasks/VsTest/distributedtest.ts

```diff
@@ -290,8 +165,8 @@ export class DistributedTest {
 
     private cleanUpDtaExeHost() {
         try {
-            if (this.testSourcesFile) {
-                tl.rmRF(this.testSourcesFile);
+            if (this.inputDataContract.TestSelectionSettings.TestSourcesFile) {
```

Comment:
do we really require as now we are using AGENT.TEMP?

Reply:
removed this